addons/calendar/models/calendar_event.py

```diff
@@ -307,15 +307,15 @@ def _inverse_dates(self):
                 # i.e. Christmas is on 25/12 for everyone
                 # even if people don't celebrate it simultaneously
                 enddate = fields.Datetime.from_string(meeting.stop_date)
-                enddate = enddate.replace(hour=18)
```

Comment:
Why this change ? I feel like the new hours are also a bit random (0 / 0), and it changes the values without much added value in my opinion. What is the issue we are trying to solve ?